src/ImageSharp/Formats/Qoi/QoiConstants.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+using System.Text;
+
+namespace SixLabors.ImageSharp.Formats.Qoi;
+
+internal static class QoiConstants
+{
+    /// <summary>
+    /// Gets the bytes that indicates the image is QOI
+    /// </summary>
+    public static ReadOnlySpan<byte> Magic => Encoding.UTF8.GetBytes("qoif");
+
+    /// <summary>
+    /// The list of mimetypes that equate to a QOI.
+    /// See https://github.com/phoboslab/qoi/issues/167
+    /// </summary>
+    public static readonly IEnumerable<string> MimeTypes = new[] { "image/qoi", "image/x-qoi", "image/vnd.qoi" };
```

Comment:
Should the type be `string[]` instead of `IEnumerable<string>`?
I.e. `string[]` allows for easier searching (if needed).

Reply:
Well I used the PngConstatns as example so I though it had to be that way, but if there's no problem I'd change it for an array